### PR TITLE
documents: add identifiedBy on some fields

### LINKED ISSUE #1924: ContentMediaCarrier : value to remove  and placeholder to add
In the professional interface,  the element Content, media, carrier type is constituted of  3 subelements : contenttype, mediatype and carriertype.
1- the value "Content type" (type de contenu) in yellow, should be removed from list of values.
2- A placeholder "Choose the type of media (choisir le type de media) should be added
image

### PR DESCRIPTION
**Pull request by Garfield-fr**
Fields involved:
supplement, supplementTo, otherEdition, otherPhysicalFormat,
issuedWith, precededBy, succeededBy, relatedTo, hasReproduction,
reproductionOf, work_access_point

* Removes templateOptions on the country jsonschema.
* Fixes typo on the type entry (lower case t).
* Adds the card wrapper on vendors jsonschema.
* Closes https://github.com/rero/rero-ils/issues/2275.
* Closes https://github.com/rero/rero-ils/issues/2388.
* Closes https://github.com/rero/rero-ils/issues/2766.
* Closes https://github.com/rero/rero-ils/issues/2659.
* Closes https://github.com/rero/rero-ils/issues/2649.
* Closes https://github.com/rero/rero-ils/issues/2340.
* Closes #1924.

⚠️  Elasticsearch reindexing.

Co-Authored-by: Bertrand Zuchuat <bertrand.zuchuat@rero.ch>

## How to test?

- Test the above mentioned fields

## Code review check list

- [ ] Commit message template compliance.
- [ ] Commit message without typos.
- [ ] File names.
- [ ] Functions names.
- [ ] Functions docstrings.
- [ ] Unnecessary commited files?
- [ ] Cypress tests successful?

### CODE REVIEW

**Review by BadrAly — APPROVED** [review]

**Review by BadrAly — APPROVED** [review]

**Comment by PascalRepond** [comment]
* [ ] `work_access_point` field contains subfields with with a `+` button. Clicking any of these button resets the field! I now have to re-select `Work (local)` to see the added fields. 
image
* [ ] The `identifier` subfield for `work_access_point` should be present by default in the editor when choosing `fwork (local)`

Mostly works as mentionned in the issues specs. Some remarks for the PO: 
* [ ] The identifier for the document in relation is searchable but it doesn't appear anywhere except in the editor. Shouldn't it be displayed in the detailed view? 
pr
* [ ] Should we really be able to add any type of identifier to `work_access_point`? It doesn't make sense to identify a work by an ISBN, ISSN, or DOI

**Comment by pronguen** [comment]
@PascalRepond 

>     * [ ]  REP/14.03: `work_access_point` field contains subfields with with a `+` button. Clicking any of these button resets the field! I now have to re-select `Work (local)` to see the added fields.
>       image

I could not reproduce. Could you specify the steps to reproduce?

>     * [ ]  REP/14.03: The `identifier` subfield for `work_access_point` should be present by default in the editor when choosing `work (local)`

Not sure we want this. Why then? This is not the case when adding a contributor (local person) or a subject (local). So I wouldn't do it differently for works (local).

**Comment by PascalRepond** [comment]
@pronguen 

> I could not reproduce. Could you specify the steps to reproduce?

1. Add `work_access_point` field to any document and select `Work (local)`
2. See that some automatically added subfields have the `+` buttons instead of empty fields (different than on test and production servers)
3. Click a `+` and see that the entire field was reset.

> Not sure we want this. Why then? This is not the case when adding a contributor (local person) or a subject (local). So I wouldn't do it differently for works (local).

OK, that is fine then.

**Review by BadrAly — APPROVED** [review]

**Review by jma — CHANGES_REQUESTED** [review]
Change your commit message "minus t" -> "lower case t"

**Review by jma — APPROVED** [review]

**Review by zannkukai — APPROVED** [review]

**Comment by PascalRepond** [comment]
https://github.com/rero/rero-ils/issues/2625 is not resolved by this PR.

File: rero_ils/modules/vendors/jsonschemas/vendors/vendor-v0.0.1.json [hunk 1]
```diff
@@ -269,6 +269,13 @@
           "type": "string",
           "format": "email"
         }
+      },
```
Comment (PascalRepond) [inline rero_ils/modules/vendors/jsonschemas/vendors/vendor-v0.0.1.json:272]: Add card wrapper to `order_contact` as well.

**Review by PascalRepond — CHANGES_REQUESTED** [review]

File: rero_ils/modules/vendors/jsonschemas/vendors/vendor-v0.0.1.json (hunk 1 above)
Comment (Garfield-fr) [reply rero_ils/modules/vendors/jsonschemas/vendors/vendor-v0.0.1.json:272]: added

**Review by PascalRepond — APPROVED** [review]

**Review by rerowep — APPROVED** [review]

**Review by BadrAly — APPROVED** [review]

**Pull request merged**